docs/docs/integrations/vectorstores/redis.ipynb

```diff
@@ -602,403 +473,570 @@
     "```\n",
     "\n",
-    "Notice that only the ``name`` field needs to be specified. All other fields have defaults."
+    "Or install and run Redis locally according to your operating system's instructions."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": null,
    "metadata": {},
    "outputs": [
     {
-     "name": "stderr",
+     "name": "stdout",
      "output_type": "stream",
      "text": [
-      "`index_schema` does not match generated metadata schema.\n",
-      "If you meant to manually override the schema, please ignore this message.\n",
-      "index_schema: {'tag': [{'name': 'credit_score'}], 'text': [{'name': 'user'}, {'name': 'job'}], 'numeric': [{'name': 'age'}]}\n",
-      "generated_schema: {'text': [{'name': 'user'}, {'name': 'job'}, {'name': 'credit_score'}], 'numeric': [{'name': 'age'}], 'tag': []}\n",
-      "\n"
+      "Connecting to Redis at: redis://redis:6379\n"
      ]
     }
    ],
    "source": [
-    "# create a new index with the new schema defined above\n",
-    "index_schema = {\n",
-    "    \"tag\": [{\"name\": \"credit_score\"}],\n",
-    "    \"text\": [{\"name\": \"user\"}, {\"name\": \"job\"}],\n",
-    "    \"numeric\": [{\"name\": \"age\"}],\n",
-    "}\n",
-    "texts = []  # list of texts\n",
-    "metadata = {}  # dictionary of metadata\n",
-    "\n",
-    "rds, keys = Redis.from_texts_return_keys(\n",
-    "    texts,\n",
-    "    embeddings,\n",
-    "    metadatas=metadata,\n",
-    "    redis_url=\"redis://localhost:6379\",\n",
-    "    index_name=\"users_modified\",\n",
-    "    index_schema=index_schema,  # pass in the new index schema\n",
-    ")"
+    "import os\n",
+    "\n",
+    "# Use the environment variable if set, otherwise default to localhost\n",
+    "REDIS_URL = os.getenv(\"REDIS_URL\", \"redis://localhost:6379\")\n",
+    "print(f\"Connecting to Redis at: {REDIS_URL}\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The above warning is meant to notify users when they are overriding the default behavior. Ignore it if you are intentionally overriding the behavior."
+    "## Importing Required Libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import wikipedia\n",
+    "from langchain.globals import set_llm_cache\n",
+    "from langchain.schema import Document\n",
+    "from langchain.text_splitter import CharacterTextSplitter\n",
+    "from langchain_core.output_parsers import StrOutputParser\n",
+    "from langchain_core.prompts import PromptTemplate\n",
+    "from langchain_core.runnables import RunnableLambda, RunnablePassthrough\n",
+    "from langchain_openai import OpenAI, OpenAIEmbeddings\n",
+    "from langchain_redis import RedisCache, RedisChatMessageHistory, RedisVectorStore"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Hybrid filtering\n",
```

Comment:
Why delete this?

Reply:
There already a section covering it, and that title was a bit misleading